ipinfo/cache.go

```diff
@@ -17,7 +17,10 @@ func NewCache(engine cache.Interface) *Cache {
 
 type EvaluatorFunc func() (interface{}, error)
 
-func (c *Cache) GetOrRequest(key string, evaluator EvaluatorFunc) (interface{}, error) {
+func (c *Cache) GetOrRequest(
```

Comment:
golint: exported method `Cache.GetOrRequest` should have comment or be unexported